Pipfile.lock

```diff
@@ -278,30 +296,37 @@
             ],
             "version": "==4.3.0"
         },
+        "pathlib2": {
+            "hashes": [
+                "sha256:8eb170f8d0d61825e09a95b38be068299ddeda82f35e96c3301a8a5e7604cb83",
+                "sha256:d1aa2a11ba7b8f7b21ab852b1fb5afb277e1bb99d5dfc663380b5015c0d80c5a"
+            ],
+            "markers": "python_version < '3.6'",
```

Comment:
there's something strange with this `markers`, they appear or disappear depending on who is running `pipenv`. Tried using https://github.com/elifesciences/digests#install-a-new-dev-package?

Reply:
I'm not sure, it could be a new thing introduced by pipenv? I don't install pipenv globally but within a virtualenv (which it complains about) but I'm likely to be using the most recent version that way. The version I generated this lockfile with is `pipenv==2018.7.1`

Reply:
maybe @de-code has some insight

Reply:
```
$ docker-compose run venv pipenv --version
Creating network "digests_default" with the default driver
Creating volume "digests_venv" with default driver
Creating volume "digests_logs" with default driver
pipenv, version 11.10.1
```
:thinking: this versioning strategy is strange

Reply:
In May the versioning was [switched to use years](https://github.com/pypa/pipenv/releases), our container version is outdated so I'll upgrade it in the base image.

Reply:
In https://github.com/elifesciences/issues/issues/4504

Reply:
thanks for that. I don't see it as a blocker to this issue, so merging.